mkdocs.yml

```diff
@@ -1,2 +1,31 @@
-site_name: DietPi-Docs
-theme: readthedocs
+site_name: DietPi Documentation
+repo_url: https://github.com/MichaIng/DietPi-Docs
+site_author: "DietPi Team"
+
+theme:
+  name: 'material'
+  #favicon: https://dietpi.com/images/logo.png
```

Comment:
This seems to be deprecated: https://github.com/mkdocs/mkdocs/commit/50e17344901579c12f1c97f4438705485f88d517
I already placed a favicon, but sadly into `img/` dir instead of into the `docs/img/`, lets move it over.